Bugzilla/Install/DB.pm

```diff
@@ -710,6 +710,9 @@ sub update_table_definitions {
     # 2014-07-27 LpSolit@gmail.com - Bug 1044561
     _fix_user_api_keys_indexes();
 
+    $dbh->bz_add_column('longdescs', 'is_markdown',
```

Comment:
Looks like most of the changes in this file have a comment referencing the bug, should we do that here too?